src/libs/getBrowser/index.web.js

```diff
@@ -18,13 +18,13 @@ function getBrowser() {
         temp = userAgent.match(/\b(OPR|Edge)\/(\d+)/);
 
         if (temp !== null) {
-            return temp.slice(1).replace('OPR', 'Opera');
+            return temp.slice(1).join(' ').replace('OPR', 'Opera');
```

Comment:
If I remember correctly, we decided we don't care about version number yet (this function is only for getting browser name). So I think we can just `return 'Opera';` here, and on line 27 below just `return 'Edge';` right?